corehq/apps/formplayer_api/smsforms/api.py

```diff
@@ -295,10 +295,12 @@ def server_down(cls):
 
 def formplayer_post_data_helper(d, content_type, url):
     data = json.dumps(d).encode('utf-8')
+    session_id = data['session-id']
```

Comment:
why does this have `session-id`, but https://github.com/dimagi/commcare-cloud/pull/3333/files#diff-d25db93c11b49b857f51d41bf25b9078R14-R16 says `sessionid` (no dash)?

Reply:
`$cookie_sessionid` is the `sessionid` cookie that commcare-hq uses in general; this 'session-id' refers to the formplayer session id, which may or may not be the same value as the `sessionid` cookie, but it actually doesn't matter for our purposes here. It gets into the request as a `X-FORMPLAYER-SESSION` header, which is referred to in nginx on the line you linked as `$http_x_formplayer_session`.

Does that answer the question? Let me know if I missed it.